### PR TITLE
Add webhook to ensure the VGStatus is active

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
**Problem:**
We need to fix the multiple times wipefs, which might break the metadata.
Also, add the webhook to ensure the VGStatus is active.

**Solution:**
add the webhook to ensure the VGStatus is active
ensure we only execute wipefs before pvcreate

**Related Issue:**

**Test plan:**

### CODE REVIEW

**Review by bk201 — APPROVED** [review]

File: pkg/provisioner/lvm.go [hunk 1]
```diff
@@ -78,6 +83,7 @@ func (l *LVMProvisioner) Provision() (bool, error) {
 	if err != nil {
 		return requeue, err
 	}
+	l.lock.Unlock()
```
Comment (tserong) [inline pkg/provisioner/lvm.go:86]: This should probably be `defer l.lock.Unlock()` and moved to immediately after line 74, else any error above will result in the lock remaining in place.

**Review by tserong — CHANGES_REQUESTED** [review]
One small suggestion, otherwise LGTM.

File: pkg/provisioner/lvm.go (hunk 1 above)
Comment (Vicente-Cheng) [reply pkg/provisioner/lvm.go]: nice catch!
I originally wanted to reduce the lock scope, so I did not use defer func (but yes, I missed the error handling)
It looks like it did not have a huge difference with/ lock the whole function, so like you mentioned, defer function would be nice.
Thanks!

**Review by tserong — APPROVED** [review]

**Comment by Vicente-Cheng** [comment]
@mergifyio backport v0.7.x

**Pull request merged**